appservice/src/connectToGitHub.ts

```diff
@@ -36,6 +37,8 @@ export async function connectToGitHub(node: AzureTreeItem, client: SiteClient, c
         const noToken: string = localize('noToken', 'No oAuth2 Token.');
         throw new Error(noToken);
     }
+    await verifyNoRunFromPackageSetting(client);
```

Comment:
I would to this after all user input, aka at the last chance possible. If they cancel at any point, we don't want to change their app settings